tests/tls_e2e/client_enc/mbedtls_client.cpp

```diff
@@ -348,6 +348,6 @@ OE_SET_ENCLAVE_SGX(
     1,    /* ProductID */
     1,    /* SecurityVersion */
     true, /* Debug */
-    512,  /* NumHeapPages */
-    128,  /* NumStackPages */
+    1024, /* NumHeapPages */
+    512,  /* NumStackPages */
```

Comment:
Do we also need to increase the stack size?

Reply:
the errors are due to Heapsize. Hence increasing heap size alone will fix the issue. The stack pages in sample applications are in the range of 1024. Hence increased it from 128 to 512 to keep it in the same order.